app/assets/stylesheets/functions/_shade.scss

```diff
@@ -0,0 +1,24 @@
+@charset "UTF-8";
+
+/// Mixes a color with black.
+///
+/// @param {Color} $color
+///
+/// @param {Number (Percentage)} $percent
+///   The amount of black to be mixed in.
+///
+/// @example scss - Usage
+///   .element {
+///     background-color: shade(#ffbb52, 60%);
+///   }
+///
+/// @example css - CSS Output
+///   .element {
+///     background-color: #664a20;
+///   }
+///
```

Comment:
Some of the other examples here just use the naked function for the example rather than including the selector and property. I think I prefer including them, but whichever way we should be consistent. Which do you prefer?